tests/lib/c_lib/src/main.c

```diff
@@ -567,9 +657,6 @@ void test_tolower_toupper(void)
 		lw[i] = tolower(test[i]);
 	}
 	lw[i] = up[i] = '\0';
-
-	zassert_equal(strcmp(up, toup), 0, "toupper error");
-	zassert_equal(strcmp(lw, tolw), 0, "tolower error");
```

Comment:
Why you remove those assertions? We need verify the API work or not after invoked.

Reply:
I move it back